sensor/kubernetes/listener/resources/virtualmachine/dispatcher/virtualmachineinstances.go

```diff
@@ -0,0 +1,99 @@
+package dispatcher
+
+import (
+	"github.com/stackrox/rox/generated/internalapi/central"
+	virtualMachineV1 "github.com/stackrox/rox/generated/internalapi/virtualmachine/v1"
+	"github.com/stackrox/rox/pkg/logging"
+	"github.com/stackrox/rox/sensor/kubernetes/eventpipeline/component"
+	"github.com/stackrox/rox/sensor/kubernetes/listener/resources/virtualmachine/store"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	kubeVirtV1 "kubevirt.io/api/core/v1"
+)
+
+var (
+	log = logging.LoggerForModule()
+)
+
+//go:generate mockgen-wrapper
+type virtualMachineStore interface {
+	Get(uid string) *store.VirtualMachineInfo
```

Comment:
```suggestion
	GetVirtualMachine(uid string) *store.VirtualMachineInfo
```

Reply:
Should we have a type for `uid` so we now that it's about id of virtual machine info

Reply:
I've added a `VMID` type